.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
Why 16.04 and not 18.04?

Reply:
Latest version Travis can provide https://docs.travis-ci.com/user/reference/overview/